dallinger/deployment.py

```diff
@@ -377,6 +377,12 @@ def setup_experiment(log, debug=True, verbose=False, app=None, exp_config=None):
             "heroku_app_id_root": six.text_type(heroku_app_id),
         }
     )
+
+    if not config.get("dashboard_password", None):
+        config.set("dashboard_password", fake.password(length=20, special_chars=False))
+    if not config.get("dashboard_user", None):
```

Comment:
The default is a random password so I don't think that's an option, we could do it for `dashboard_user` I guess.

Reply:
`dashboard_user` was the value I was thinking of. I don't feel super-strongly about it, but I think it would be good to prevent a gradual slide back into sprinkling config defaults around everywhere.